drivers/ethernet/dsa_ksz8794.c

```diff
@@ -0,0 +1,413 @@
+/*
```

Comment:
change the header of your commit title to "drivers: ethernet: ..."

Reply:
OK